main/src/cgeo/geocaching/CompassActivity.java

```diff
@@ -408,7 +409,7 @@ private void updateDeviceHeadingAndOrientation(final DirectionData dir) {
         while (direction >= 360f) {
             direction -= 360f;
         }
-        deviceHeading.setText(String.format("%3.1f°", direction));
+        deviceHeading.setText(String.format(Locale.US, "%3.1f°", direction));
```

Comment:
Shouldn't we use the user's locale here (maybe default locale)?
Using Locale.US would format decimal numbers with a decimal point instead.

Reply:
That would have been my original intention. But if language specific separators are preferred that's fine for me too.

Reply:
1:1
And I would leave the decition to you ...
Ok, I will change it.

Reply:
Thanks. If I was looking at the right part of the dialog, it's localized in current release (probably due to implicit use of default locale).

Reply:
done